recipes/dustmaps/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "dustmaps" %}
+{% set version = "0.1a12" %}
+{% set sha256 = "2018dc55c9ea3b9c7c1336b102eb5f75708a0c3de4a14d00b43de3c25c0b5766" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - pip
+    - numpy
+    - scipy
+    - astropy
+    - h5py
+    - healpy
+    - requests
+    - progressbar2
+    - six
+  run:
+    - python
+    - numpy
+    - scipy
+    - astropy
+    - h5py
+    - healpy
+    - requests
+    - progressbar2
+    - six
+
+test:
+  imports:
+    - dustmaps
+
+about:
+  home: https://github.com/gregreen/dustmaps
+  license: GPL-2.0
+  license_file: LICENSE.txt
+  summary: 'A uniform interface for a number of 2D and 3D maps of interstellar dust reddening/extinction.'
+
+  doc_url: http://dustmaps.readthedocs.io/
+  dev_url: https://github.com/gregreen/dustmaps
+
+extra:
+  recipe-maintainers:
+    - guillochon
+    - pkgw
```

Comment:
@pkgw are you interested in being a maintainer?

Reply:
Whoops, this got a bit stale! Yes, I am.